TriblerGUI/widgets/trustpagegraph.py

```diff
@@ -69,7 +69,7 @@ def compute_initial_figure(self):
         self.draw()
 
 
-class TrustPage(QWidget):
+class TrustPageGraph(QWidget):
```

Comment:
I think this name is slightly confusing since you're also displaying a graph. Maybe we should rename this class to something more distinct.